types/linkifyjs/linkifyjs-tests.tsx

```diff
@@ -225,3 +227,17 @@ describe("linkifyjs/react", () => {
         </Linkify>;
     }
 });
+
+describe("linkifyjs/string", () => {
```

Comment:
Since the tests were based off `https://soapbox.github.io/linkifyjs/docs/linkify-string.html`, it might be a good idea to link back to it.